src/app/vendors/SingleVendor.tsx

```diff
@@ -177,11 +178,12 @@ function SingleVendor({
     if (artifactDisplay) {
       displayName = artifactDisplay.name;
       displayDesc = artifactDisplay.description;
+      artifactCheck = 'seasonalArtifact';
     }
   }
 
   return (
-    <div className={clsx(styles.page, 'dim-page')}>
+    <div className={clsx(styles.page, 'dim-page')} id={artifactCheck}>
```

Comment:
I'd prefer if you used a class instead of an ID

Reply:
This is a class now. 
Also changed the empty state for non-artifact back to `let artifactCheck = "";` because the other way way puting an `undefined` string the class (formerly ID) when set as `let artifactCheck:string|undefined;`